js/views/settingsVw.js

```diff
@@ -635,6 +676,7 @@ module.exports = Backbone.View.extend({
       }
     });
 
+    this.blockedUsersVw.remove();
```

Comment:
Another refactor candidate...
The functionality that deals with subViews would be a good candidate to go inside a 'base' view that every view would extend from. The way it is now, every view is manually creating their own functionality for this (which are mostly copies of each other) and that can be difficult to manage.

I have a base view that I've used on previous projects that implements some child management functionality that is pretty simple to use and has worked well for me. If we want, at some point, I could bring it in here. The main focus of it is it handle's the removal of child views when you remove a parent view, thus preventing ghost views (i.e. memory leaks).